<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="PageCreate">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>password   </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textPassword"/>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Name         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textName"/>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="btnConfirm">
            <property name="text">
             <string>Confirm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnListPage_KeyMember">
            <property name="text">
             <string>List Key Member Only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnListPage">
            <property name="text">
             <string>List All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>username  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textUsername"/>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>phone        </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textPhone"/>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>products</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QCheckBox" name="checkBoxStandardAdd">
             <property name="text">
              <string>standard</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxPremiemAdd">
             <property name="text">
              <string>premiem</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxProAdd">
             <property name="text">
              <string>pro</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Address     </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textAddress"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>credit         </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textCredit"/>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <widget class="QRadioButton" name="radioButtonKeyCreate">
          <property name="text">
           <string>Key Member</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="PageList">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="5" column="0" colspan="3">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="labelAddressListPage">
              <property name="text">
               <string>Address</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textAddressListPage"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="labelRatingListPage">
              <property name="text">
               <string>rating    </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="textRatingListPage"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9"/>
          </item>
         </layout>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="btnDelete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QPushButton" name="btnUpdate">
          <property name="text">
           <string>Update</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="labelCustomerName">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="3">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="title">
           <string>Products</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_13">
           <item>
            <widget class="QCheckBox" name="checkBoxStandardUpdate">
             <property name="text">
              <string>Standard: $100</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxPremiemUpdate">
             <property name="text">
              <string>Premiem: $200</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxProUpdate">
             <property name="text">
              <string>Pro: $300</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="btncreatePage">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2" colspan="3">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" rowspan="2" colspan="2">
           <widget class="QTableView" name="tableView">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>200</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QRadioButton" name="radioButtonKeyUpdate">
          <property name="text">
           <string>Key Member</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
